pkg/daemon/ceph/osd/volume.go

```diff
@@ -628,7 +630,7 @@ func (a *OsdAgent) initializeDevicesLVMMode(context *clusterd.Context, devices *
 					metadataDevices[md]["devices"] = deviceArg
 				}
 				deviceDBSizeMB := getDatabaseSize(a.storeConfig.DatabaseSizeMB, device.Config.DatabaseSizeMB)
-				if storeFlag == "--bluestore" && deviceDBSizeMB > 0 {
+				if (storeFlag == "--bluestore" || storeFlag == "--bluestore_rdr") && deviceDBSizeMB > 0 {
```

Comment:
Can we change this to use the API-defined constants here so we aren't creating hardcoded versions of the same constants in a different place in the code?

I think this should work:

```go
storeType := a.storeConfig.GetStoreType()

# ...

if (storeType == cephv1.BlueStore || storeType == cephv1.BlueStoreRDR) && deviceDBSizeMB > 0 {
   # ...

```

Alternately, we could make an API method that would return true if it's bluestore or bluestore_rdr like `a.storeConfig.IsBlueStoreOrRDR()`